modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1454,7 +1454,7 @@ SHOW INDEXES YIELD *
 =====
 
 [#_neo_clientnotification_statement_runtimeexperimental]
-=== RuntimeExperimental
+=== RuntimeExperimental  
```

Comment:
This notification is no longer in use and doesn't have any GQL status, hence no new heading